packages/library/src/transaction-confirmation.ts

```diff
@@ -73,7 +77,18 @@ export function createDefaultRecentTransactionConfirmer({
     rpc,
     rpcSubscriptions,
 }: DefaultRecentTransactionConfirmerConfig) {
-    const getBlockHeightExceedencePromise = createBlockHeightExceedencePromiseFactory(rpcSubscriptions);
+    const getSleepDelayBetweenBlockHeightFetches = getDefaultSleepDelayBetweenBlockHeightFetchesGetter(
+        400 /* msPerSlot */,
```

Comment:
I really hate hardcoding this, and I'm open to suggestions.

Reply:
May I ask the rationale behind that number?

Reply:
The block height can only possibly advance once per slot. 400ms is the current target latency for slot progression.

If any given slot is skipped, the block height will not advance. This is why the block height is nearly always lower than the slot height.